core/DataAccess/LogQueryBuilder/JoinTables.php

```diff
@@ -48,7 +48,22 @@ public function addTableToJoin($tableName)
 
     public function hasJoinedTable($tableName)
     {
-        return in_array($tableName, $this->getTables());
+        $tables = in_array($tableName, $this->getTables());
+        if ($tables) {
```

Comment:
hasJoinedTable was incorrect and did not look at tables that are defined via an array...